PLUB/Sources/Views/Home/MeetingSchedule/Cell/ScheduleTableViewCell.swift

```diff
@@ -114,8 +123,15 @@ final class ScheduleTableViewCell: UITableViewCell {
       $0.leading.equalToSuperview().inset(17)
```

Comment:
저희 양쪽 패딩은 16으로 동일해요! 수정해주세요 :)
```suggestion
      $0.leading.equalToSuperview().inset(16)
```